### DISCUSSION TITLE
AIR Applications look blurry on 2K displays

### DISCUSSION THREAD

**Discussion by ylazy** [airsdk/Adobe-Runtime-Support#1054] [General]
My AIR application looks blurry on 2K displays.
Export Settings:
Window Styles: Custom Chrome (opaque)
Profiles: Extended Desktop.
2K Display:
![bug-2k](https://user-images.githubusercontent.com/6010183/128658155-b9d0e80e-0c43-4323-a3c8-35d7cf369b16.png)
HD Display:
![normal-display](https://user-images.githubusercontent.com/6010183/128658158-dc7472cf-d7a5-4e1c-8f44-c72b46ba5d00.png)
Please help!
Thanks!

**Comment by ajwfrost** [comment]
Hi
In your app descriptor file, in the `<initialWindow>` section, do you have the setting:
```
<requestedDisplayResolution>high</requestedDisplayResolution>
```
thanks

**Reply by ylazy** [reply]
Wow! It worked. Don't understand why I didn't know this for years. Thank you so much!
